oak_restricted_kernel_runtime/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "oak_restricted_kernel_runtime"
```

Comment:
A bit of bikeshedding: I am not sure that "runtime" is the correct name here, but don't really have a better suggestion. "lib" is an obvious candidate, but very vague. "sdk" was another option, but seems a bit inaccurate.

Reply:
I also don't think runtime is accurate. How about `lib` or `sys`? `sdk` is also fine.

Reply:
I think `lib` might be confusing in Rust, (since there are `lib.rs` files), and the `oak_restricted_kernel` crate does have a `lib.rs` already: https://github.com/project-oak/oak/blob/main/oak_restricted_kernel/src/lib.rs

Could it be named `oak_restricted_kernel_api` or `_abi` (not sure which one is correct here) ?
Also like the `sdk`

Reply:
Here's an option to do more bikeshedding :)

I split the crate into two crates:
  - `oak_restricted_kernel_interface` that just contains the basic data structures (akin to linux kernel headers); the idea is that this is the interface that both the kernel and the user code can rely on
  - `oak_restricted_kernel_api` is the actual Rust-y user space interface implementation that relies on the interface.